docs/links/github_pr_6090.py

```diff
@@ -0,0 +1,7 @@
+from . import link
+
+link_name = "github_pr_6090" 
+link_text = "github_pr_6090"
```

Comment:
```suggestion
link_text = "GitHub PR #6090"
```

I would prefer to make these human-readable.